Library/Homebrew/extend/ENV.rb

```diff
@@ -33,19 +33,30 @@ def activate_extensions!(env: nil)
 
   sig {
     params(
-      env:          T.nilable(String),
-      cc:           T.nilable(String),
-      build_bottle: T::Boolean,
-      bottle_arch:  T.nilable(String),
-      _block:       T.proc.returns(T.untyped),
+      env:           T.nilable(String),
+      cc:            T.nilable(String),
+      build_bottle:  T::Boolean,
+      bottle_arch:   T.nilable(String),
+      debug_symbols: T.nilable(T::Boolean),
+      _block:        T.proc.returns(T.untyped),
     ).returns(T.untyped)
   }
-  def with_build_environment(env: nil, cc: nil, build_bottle: false, bottle_arch: nil, &_block)
+  def with_build_environment(
+    env: nil,
+    cc: nil,
+    build_bottle: false,
+    bottle_arch: nil,
+    debug_symbols: false,
+    &_block
+  )
```

Comment:
Keep these in as few lines as `rubocop` will allow.

Reply:
Done in cd9fe97c551d4e9b5f0a702a1382a6ccbe494773